code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -62,13 +62,14 @@
 	load_method = MAGAZINE
 	magazine_type = /obj/item/ammo_magazine/m545
 	allowed_magazines = list(/obj/item/ammo_magazine/m545)
+	projectile_type = /obj/item/projectile/bullet/rifle/a545
 
-	one_handed_penalty = 4
+//	one_handed_penalty = 30

```

Comment:
4*15=60?

Reply:
I felt that 60 was a bit much, honestly. This is an antag-only assault rifle, I feel like they're allowed to cheat a little. Not married to it, though.